src/runtime/storageNG/tests/store-test.ts

```diff
@@ -130,6 +131,7 @@ describe('Store', async () => {
     const store = new Store(testKey, Exists.ShouldCreate, null, StorageMode.Direct, CRDTCount);
     const activeStore = await store.activate();
     
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
You can remove this line by removing `async` in the promise executor by removing `await` at its last line, i.e.:

Just do `activeStore.onProxyMessage({type: ProxyMessageType.SyncRequest, id: id1});`, without the await, the result will be the same.

Reply:
done